### ISSUE TITLE
Final Peer Review

### ISSUE THREAD

**Issue by joeldsouza493** [HehongLi08/WhichHospitalToVisit#12]
The project aims to come up with a model to recommend hospitals for patients. The dataset used for this project is the Statewide Planning and Research Cooperative System (SPARCS) Inpatient De-Identified dataset.

Things I like:
1. The team has put a lot of effort into data cleaning and feature engineering methods. This helped them get good results.
2. The report mentions the reasons for model selection. Also, the results are presented well in the report
3. The team has put forth an idea as to how they can extend this project to an application in the production level.
4. The fairness of the model was presented well in the video

Notes:
1. Only linear models are used for cost prediction. I believe that analysis with complex models could have yielded better results.
2. To identify which hospital to visit, one must consider other factors like quality of treatment, success rate, or other proxies along with the cost. The team has mentioned it in the future scope, but I think it could have made the analysis interesting if included in the current scope.
3. The Ridge regression and quantile regression are compared but trained on a different set of features. 
4. The report does not mention the validation set or technique for selecting the regularization parameter for ridge regression. It is making the decision based on test set error that makes the result biased.
